src/main/controllers/StepsToMakingYourClaimController.ts

```diff
@@ -27,6 +27,7 @@ export default class StepsToMakingYourClaimController {
       const redisClient = req.app.locals.redisClient;
       const caseData = await getPreloginCaseData(redisClient, req.session.guid);
       if (userCase.id === undefined) {
+        // todo try-catch this - if createCase errors the whole app fails.
```

Comment:
todo - but not in this PR?

Reply:
I got carried away with a few things but at the end of the day it is a logging-improvement PR, I couldn't clean it all.

Reply:
That's fair, suppose it should be a wider discussion what we do if this call fails, like you say, it's critical for the entire app to work

Reply:
I'll raise a ticket for this, really this isn't for this ticket to deal with this.